### DISCUSSION TITLE
EVU Sperre

### DISCUSSION THREAD

**Discussion by AISMap** [pail23/stiebel_eltron_isg_component#155] [Q&A]
Hi Paul,

is there a way to find out whether the heat pump is operational or power supply is switched off by the power station ("EVU Sperre") ?

**Comment by pail23** [comment]
Hi @AISMap 

There is a register 2502 on the modbus protocol called "EVU-FREIGABE" (see [here](https://www.stiebel-eltron.ch/content/dam/ste/ch/de/downloads/kundenservice/smart-home/Modbus/Modbus%20Bedienungsanleitung.pdf), page 7). Maybe you coud try to read this with the [home assistant modbus](https://www.home-assistant.io/integrations/modbus/) implementation. In case it works for you, I'm happy to add it to the Stiebel Eltron component.

**Comment by dickesW** [comment]
Added the feature here:
https://github.com/pail23/stiebel_eltron_isg_component/pull/347
